### ISSUE TITLE
Relation with composable.el

### ISSUE THREAD

**Issue by DamienCassou** [cute-jumper/embrace.el#6]
I have the impression that there are some relationships with [`composable.el`](https://github.com/paldepind/composable.el). For example, you could sync with composable's author to agree on which character selects which unit/object. You seem to agree on some already, e.g., `w` for `word`.

**Comment by cute-jumper** [comment]
Yeah. I also like `composable.el`. For `embrace-add`, it is really similar to the composable commands in `composable.el` although there are some differences. I'll take a look at this issue a little bit later.
